packages/AlgernonTests.package/ALGTestTypes.class/instance/testPackageAction.st

```diff
@@ -5,5 +5,5 @@ testPackageAction
 	self moveCursorTo: 'Package'.
```

Comment:
wasn't there a helper for `runQuery .. withCursorAt` 🤔

Reply:
I don't see it? What would it be called?

Reply:
`runQuery: query withCursorAt: aCategory` could be used to refactor line 4-5 if I am not mistaking.

Reply:
I didn't get your question and didn't look at the code, I thought you meant `runQuery:withCursorAt:` is already implemented.